third-party/qthread/Makefile

```diff
@@ -132,6 +132,11 @@ ifneq (, $(findstring pgi,$(CHPL_MAKE_TARGET_COMPILER)))
 CHPL_QTHREAD_CFG_OPTIONS += --disable-internal-spinlock
 endif
 
+# if we are using the bundled hwloc then we must specify the prefix for its symbols
+ifeq ($(CHPL_MAKE_HWLOC),bundled)
```

Comment:
Maybe check for nonblank `CHPL_HWLOC_PREFIX` here instead of bundled hwloc? Seems more direct.

Reply:
Somewhere I have to check for bundled hwloc to know whether or not to set the CHPL_HWLOC_PREFIX. I could do that in hwloc/Makefile.in if you think that's a better solution.

Reply:
If we're not using a bundled hwloc then `CHPL_HWLOC_PREFIX` will not be set and `$(CHPL_HWLOC_PREFIX)` will just be empty. But to make it more obvious we could do `ifndef CHPL_HWLOC_PREFIX` instead of checking for an unset value.